glue/src/it/scala/com/gu/tableversions/glue/GlueMetastoreIntegrationSpec.scala

```diff
@@ -6,27 +6,17 @@ import cats.effect.IO
 import cats.implicits._
 import com.amazonaws.auth._
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
-import com.amazonaws.services.glue.model.{
-  Column,
-  CreateTableRequest,
-  DeleteTableRequest,
-  GetPartitionsRequest,
-  GetTableRequest,
-  SerDeInfo,
-  StorageDescriptor,
-  TableInput
-}
+import com.amazonaws.services.glue.model._
 import com.amazonaws.services.glue.{AWSGlue, AWSGlueClient}
 import com.gu.tableversions.core.Partition.PartitionColumn
-import com.gu.tableversions.core._
-import com.gu.tableversions.metastore.MetastoreSpec
-import org.scalatest.{Assertion, BeforeAndAfterAll, FlatSpec, Matchers}
+import com.gu.tableversions.core.{MetastoreSpec, _}
+import org.scalatest.{Assertion, FlatSpec, Matchers}
 
 import scala.util.{Properties, Random}
 
-class GlueMetastoreSpec extends FlatSpec with Matchers with MetastoreSpec {
+class GlueMetastoreIntegrationSpec extends FlatSpec with Matchers with MetastoreSpec {
```

Comment:
👍 I was considering renaming this myself but then I forgot to do it

Reply:
I only did it because IntelliJ complained about it...